apps/api/src/services/amazon/s3.py

```
@@ -40,11 +40,13 @@ async def download_and_upload(url: str, referer: str = None, suite: str = "test"
 
 async def upload(
         staging_path: Path,
-        bucket: str = AWS_S3_BUCKET_NAME,
+        bucket: str = None,
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)